deploy/group_vars/all/main.yml

```diff
@@ -222,6 +222,11 @@ deploy_user: vagrant
 pp_dba_username: vagrant
 pp_dba_password: vagrant
 
+# Replication user
+replication_user: replication
+replication_password: replication
+
+pp_datadir: /var/lib/pgsql/10/data
```

Comment:
@rioh Could you also remove these from here?  That way they are not set in multiple locations, which could lead to confusion down the road.

Reply:
@tanderegg Turns out I did need those in group_vars.  I have a play within deploy_citus_server.yml that only has the postgresql-server-config role which don't have those variables defined.  Maybe add postgresql as a dependency for postgresql-server-config?

Reply:
@rioh Actually, I think it makes sense to just copy them to postgresql-server-config.  Having postgresql-server as a dependency would cause it to run whenver postgresql-server-config is run, which isn't always desirable.